Day01/app.py

```diff
@@ -1,42 +1,30 @@
-import itertools as it
 from pathlib import Path
 
 # Creo un oggetto Path
 path = Path("Day01/input.txt")
 
-
-# creo una lista per verificare se un numero è presente nelle righe
-lista_numeri: list[int] = [numero for numero in range(1, 2021)]
-
-
 # Apro il file in lettura
 with open(path, "r") as file:
-    riga = file.readline()
```

Comment:
Questo readline() era il problema che rendeva il risultato leggeremente sbagliato. 
In pratica se dopo aver aperto il file con `open()` chiami subito `readline()` il puntatore del file passa subito alla seconda riga.
Infatti nel risultato mancava la somma della prima riga perchè non la leggevamo proprio